pkg/env/max_parallel_scans.go

```diff
@@ -4,4 +4,8 @@ var (
 	// MaxParallelImageScanInternal sets the max number of parallel scans collectively on the ScanImageInternal, EnrichLocalImageInternal,
 	// and GetImageVulnerabilitiesInternal endpoints and separately sets the max active local scans in a secured cluster
 	MaxParallelImageScanInternal = RegisterIntegerSetting("ROX_MAX_PARALLEL_IMAGE_SCAN_INTERNAL", 30)
+
+	// MaxParallelAdHocScanInternal defines the maximum number of parallel ad-hoc roxctl delegated scans initiated from Central.
+	// It must be set lower than the collective limit for ScanImageInternal, EnrichLocalImageInternal, and GetImageVulnerabilitiesInternal endpoints.
+	MaxParallelAdHocScanInternal = RegisterIntegerSetting("ROX_MAX_PARALLEL_ADHOC_SCAN_INTERNAL", 5)
```

Comment:
This is not an 'Internal' limit like the other one so can remove `Internal' from the name.

```suggestion
	// MaxParallelAdHocScan defines the maximum number of parallel ad-hoc delegated scans.
	// It must be set lower than MaxParallelImageScanInternal
	MaxParallelAdHocScans = RegisterIntegerSetting("ROX_MAX_PARALLEL_ADHOC_SCAN", 5)
```

Also wondering if we should make this a sensor specific env variable, ie: put in `env/sensor.go` and name something similar to `ROX_SENSOR_MAX_PARALLEL_ADHOC_SCAN`, unless we foresee needing this in Central.